conf/playReleases.json

```diff
@@ -1,18 +1,19 @@
 {
     "latest": {
-        "version": "2.3.9",
-        "date": "May 9 2015"
+        "name": "Damiya",
+        "version": "2.4.0",
+        "date": "May 26 2015"
     },
     "development": [
         {
-            "version": "2.4.0-RC5",
+            "version": "3.0.0-SNAPSHOT",
```

Comment:
We don't usually publicise nightly builds, only milestones and releases, when there's no current milestone/release, we make this an empty list.